examples_tests/__init__.py

```diff
@@ -0,0 +1,161 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015, 2016 Canonical Ltd
```

Comment:
The file is new, but I started writing this code last year. I'm just refactoring it now. So as I understand it, it should have both years.